test/tls/web-config.yml

```diff
@@ -0,0 +1,5 @@
+# Minimal TLS configuration example. Additionally, a certificate and a key file
```

Comment:
No need to include this file if it's described in the README.

Reply:
Done